### PR TITLE
MRG: Powerlaw estimation updated

### PR DESCRIPTION
**Pull request by z357412526**
Fix the format and add functions.

### CODE REVIEW

**Comment by CenzhuoYao** [comment]
It seems good and ready to merge now.

File: mousestyles/est_power_param.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    --------
+    >>> fit_exponential (0, 0, 0)
+    7.385844980814098
+    """
+    cut_dist = getdistance(strain, mouse, day)
+    ret_mle = len(cut_dist) / (np.sum(cut_dist) - len(cut_dist))
+    return ret_mle
+
+
+def fit():
```
Comment (hduongtrong) [inline mousestyles/est_power_param.py]: maybe a more explicit function name, e.g. fit_dist_all

**Comment by lunadadada** [comment]
Reviewed and need this to be merged since my next step depends on her functions.

**Comment by jarrodmillman** [comment]
@z357412526 this looks good.  I made a new pr #174 by cherry-picking your commits so I could rebase your work and clean up your history.  As soon as it finishes running the tests, I will merge it and close this pr.

**Pull request closed without merging**